conf/timezones.ics

```diff
@@ -1919,11 +1937,11 @@ END:DAYLIGHT
 END:VTIMEZONE
 BEGIN:VTIMEZONE
 TZID:America/Caracas
-LAST-MODIFIED:20170601T083716Z
+LAST-MODIFIED:20170711T043529Z
 X-ZIMBRA-TZ-PRIMARY:TRUE
 X-ZIMBRA-TZ-MATCH-SCORE:100
-X-ZIMBRA-TZ-ALIAS:(GMT-04.30) Caracas
-X-ZIMBRA-TZ-ALIAS:(UTC-04.30) Caracas
+X-ZIMBRA-TZ-ALIAS:(GMT-04.00) Caracas
```

Comment:
@rupalid @silentsakky The idea for timezone aliases, rightly or wrongly is to map out of date timezones to the most appropriate current timezone.  Given that, it is useful to have aliases with the wrong offset in the description, otherwise they get orphaned with outdated rules.